dev-docs/RFCs/v7.0/google-maps-integration.md

```diff
@@ -12,12 +12,12 @@ This RFC proposes a new module `@deck.gl/google-maps` that handles the synchroni
 
 ## Proposal
 
-### DeckOverlay
+### GoogleMapsOverlay
```

Comment:
Is this changed based on the suggestion I made in my doc, or other reasoning? I've been waffling back and forth on this.

Reply:
The master bundle (deck.gl's dist.min.js) will include all exports under the global `deck` object, e.g. `Deck`, and all the layers. `DeckOverlay` in that context becomes very confusing.